### ISSUE TITLE
dice coeff or loss function gathering

### ISSUE THREAD

**Issue by cmlyldz** [milesial/Pytorch-UNet#99]
Can I get loss function and dice coefficent from checkpoint.pth and/or model.pth?

**Comment by milesial** [comment]
Hi, you can find the dice coefficient I got in the readme
